product_docs/docs/eprs/7/05_smr_operation/08_optimizing_performance/02_optimize_sync_replication/using_bacth_lob.mdx

```diff
@@ -16,7 +16,7 @@ You can configure the replication of up to 1000 rows per batch.
 
 ### Prerequisites 
 
-You have upgraded your instances to a Replication Server version that includes this feature. See [Release Notes](../../../eprs_rel_notes) for an overview of the available release versions and the included enhancements. 
+Upgrade your instances to a Replication Server version that includes this feature. See [Release notes](../../../eprs_rel_notes) for an overview of the available release versions and the included enhancements. 
```

Comment:
@ebgitelman I usually use the past form in prerequisites sections because the items mentioned here must have already happened for the reader to be able to perform the steps on this page. 

I am not against using the present here, just explaining what I did here in case it changes your mind. Let me know what you think, so I can keep that in mind for the future :).

Reply:
I see what you're saying but it poked me in the eye as not quite the right tense here. This could be:
Your instances must be upgraded... 
or really, what you want to say is
Your instances must use a Replication Server version...

That definitely gives more of a sense of prerequisite. If you like that, can you make that change before you approve ithis?